lambdas/pkgpush/src/t4_lambda_pkgpush/__init__.py

```diff
@@ -148,6 +162,57 @@ def calculate_pkg_hashes(pkg: quilt3.Package):
             f.result()
 
 
+def invoke_copy_lambda(credentials: AWSCredentials, src: PhysicalKey, dst: PhysicalKey) -> str:
+    result = invoke_lambda(
+        function_name=S3_COPY_LAMBDA,
+        params=S3CopyLambdaParams(
+            credentials=credentials,
+            location=S3ObjectSource.from_pk(src),
+            target=S3ObjectDestination.from_pk(dst),
+        ),
+        err_prefix="S3CopyLambda",
+    )
+    return CopyResult(**result).version
+
+
+def copy_pkg_entry_data(
+    credentials: AWSCredentials,
+    src: PhysicalKey,
+    dst: PhysicalKey,
+    idx: int,
+) -> T.Tuple[int, PhysicalKey]:
+    version_id = invoke_copy_lambda(credentials, src, dst)
+    return idx, PhysicalKey(bucket=dst.bucket, path=dst.path, version_id=version_id)
+
+
+def copy_file_list(
+    file_list: T.List[T.Tuple[PhysicalKey, PhysicalKey, int]],
+    message=None,
+    callback=None,
+) -> T.List[PhysicalKey]:
+    # TODO: Copy single part files directly, because using lambda for that just adds overhead,
```

Comment:
@nl0 
I don't want to use `XXX` because I find it quite unclear. By [wikipedia](https://en.wikipedia.org/wiki/Comment_(computer_programming)#Tags) its means
> warn other programmers of problematic or misguiding code

I agree that `TODO` is not quite clear in terms whether it should be done in scope of this PR or in future. (or it is if compare with `FIXME`)?

So maybe I should start using `FUTURE` for improvements to be done in bright future?